runtime/fundamentals/modules.md

```diff
@@ -0,0 +1,397 @@
+| `>=1.2.3` | Greater than or equal to version 1.2.3. Any version `1.2.3` or higher is allowed.                                                                                   | `>=1.2.3` |
+| `<=1.2.3` | Less than or equal to version 1.2.3. Any version `1.2.3` or lower is allowed.                                                                                       | `<=1.2.3` |
+| `>1.2.3`  | Greater than version 1.2.3. Only versions higher than `1.2.3` are allowed.                                                                                          | `>1.2.3`  |
+| `<1.2.3`  | Less than version 1.2.3. Only versions lower than `1.2.3` are allowed.                                                                                              | `<1.2.3`  |
+| `1.2.x`   | Any patch version within the minor version 1.2. For example, `1.2.0`, `1.2.1`, etc.                                                                                 | `1.2.x`   |
+| `1.x`     | Any minor and patch version within the major version 1. For example, `1.0.0`, `1.1.0`, `1.2.0`, etc.                                                                | `1.x`     |
+| `*`       | Any version is allowed.                                                                                                                                             | `*`       |
+
+## URL imports
+
+Deno also supports import statements that reference URLs, either directly:
+
+```js
+import { Application } from "https://deno.land/x/oak/mod.ts";
+```
+
+or part of your `deno.json` import map:
+
+```json
+{
+  "imports": {
+    "oak": "https://deno.land/x/oak/mod.ts"
+  }
+}
+```
+
+Supporting URL imports enables us to support the following JavaScript CDNs, as
+they provide URL access to JavaScript modules:
+
+- [deno.land/x](https://deno.land/x)
+- [esm.sh](https://esm.sh)
+- [unpkg.com](https://unpkg.com)
+
+URL imports are useful if you have a small, often single file, Deno project that
+doesn't require any other configuration because you can avoid having a
+`Deno.json` file at all. It's not advised to use this style of import in larger
+applications as you may end up with version conflicts (where different files use
+different version specifiers).
+
+:::info
+
+Use URL imports with caution, and only **from trusted sources**. If the server
+is compromised, it could serve malicious code to your application. They can also
+cause versioning issues if you import different versions in different files. URL
+imports remain supported, **but we recommend using a package registry for the
+best experience.**
+
+:::
+
+### Overriding URL imports
+
+The other situation where import maps can be very useful is to override URL
+imports in specific modules.
+
+Let's say you want to override a `https://deno.land/x/my-library@1.0.0/mod.ts`
```

Comment:
can we use a jsr example rather than deno.land?

Reply:
URL imports cannot be overridden with the `scopes` import feature afaik